vochain/start.go

```diff
@@ -361,3 +379,9 @@ func AminoKeys(key crypto25519.PrivKey) (private, public []byte, err error) {
 
 	return aminoKey, aminoPubKey, nil
 }
+
+// AminoPubKey is a helper function that transforms a standard EDDSA pubkey into
+// Tendermint like amino format useful for creating genesis files.
+func AminoPubKey(pubkey []byte) (public []byte, err error) {
```

Comment:
As public and err are not assigned here you can just have `func AminoPubKey(pubKey []byte) ([]byte, error)`